layouts/trackers/single.html

```diff
@@ -98,7 +102,7 @@ <h2>{{ T "tracker-single-transmitted-data-heading" }}</h2>
                                         If the reasoning contains an external link, this creates an link to that, if it references a Markdown file in `research`, it links to the corresponding section.
                                     -->
                                     {{ if or (strings.HasPrefix .reasoning "https://") (strings.HasPrefix .reasoning "http://") }}
-                                        <a href="{{ .reasoning }}"><img src="/svg/external.svg" class="inline-icon icon-external" alt="{{ T "tracker-single-transmitted-data-table-reasoning-external-link-desc" }}" title="{{ T "tracker-single-transmitted-data-table-reasoning-external-link-desc" }}"></a>
+                                        <a href="{{ .reasoning }}"><img src="/svg/external.svg" class="inline-icon icon-external" alt="{{ T "tracker-single-transmitted-data-table-reasoning-external-link-desc" }}" title="{{ T "tracker-single-transmitted-data-table-reasoning-external-link-desc" }}" rel="nofollow"></a>
```

Comment:
I feel like the `nofollow` would make more sense on the link, rather than the image…